tutorials/hyperspectral/Advanced_Spectral_Signatures.Rmd

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
Can you rearrange to put syncID into the 2nd row for consistency with other tutorials?

Reply:
Yup! Done.